molecule/default/molecule.yml

```diff
@@ -6,15 +6,13 @@ driver:
 lint: |
   set -e
   yamllint .
+  ansible-lint
```

Comment:
Should this also be included on line 8 of the kind/molecule.yml file?

https://github.com/ansible/awx-operator/pull/1101/files#diff-0aa6fe9c9b445b24185faa9e2ecae5ed7a4829e04bc50cecf8e7fc9c0fb9dadeR8

Reply:
looks like it fails when put in the kind/molecule.yml.

Reply:
What kind of failures have you seen ? IMHO we should have the same change in the `kind/molecule.yml` file too, I just missed to added it to that file

Reply:
Using `community.docker.docker_image` didn't seem to fix it...

Reply:
I don't see molecule using the kustomize.yml file so I'm wondering if it is required at all and we could remove it. But if it is required maybe we can exclude it from the ansible-lint configuration since ansible-lint wants that file to the be playbook rather than a tasks file. 

the second issue is probably because you didn't install the collection from https://github.com/ansible/awx-operator/blob/devel/molecule/requirements.yml

Reply:
> I don't see molecule using the kustomize.yml file

```console
$ grep -r 'kustomize.yml' .
./molecule/default/converge.yml:    - import_tasks: kustomize.yml
./molecule/default/destroy.yml:    - import_tasks: kustomize.yml
```
So the file is used, we might want to skip it in the ansible-lint config or move it to another directory so ansible-lint won't consider it as a playbook.

Reply:
wow after moving the kustomize.yml file to a dedicated tasks directory then ansible-lint is now reporting a lot of things...

```console
                              Rule Violation Summary                              
 count tag                    profile    rule associated tags                     
    26 key-order[task]        basic      formatting, experimental (warning)       
    21 jinja[spacing]         basic      formatting (warning)                     
     4 schema[meta]           basic      core, experimental (warning)             
     3 schema[vars]           basic      core, experimental (warning)             
    34 name[missing]          basic      idiom                                    
     3 name[play]             basic      idiom (warning)                          
     3 name[template]         moderate   idiom                                    
     2 risky-file-permissions safety     unpredictability, experimental (warning) 
    11 no-changed-when        shared     command-shell, idempotency               
   204 fqcn[action-core]      production formatting                               
   100 fqcn[action]           production formatting (warning)                     

Failed after min profile: 252 failure(s), 159 warning(s) on 191 files.
```